test-suite/misc/poly-capture-global-univs.sh

```diff
@@ -7,7 +7,7 @@ export PATH=$COQBIN:$PATH
 
 cd misc/poly-capture-global-univs/
 
-coq_makefile -f _CoqProject -o Makefile
+coq_makefile -coqlib "$LIB"/../.. -f _CoqProject -o Makefile
```

Comment:
coq_makefile doesn't understand -coqlib
Also why ../.. as in my other comment

Reply:
It doesn't understand it but changes in 2.0 make it use the coqlib infrastructure [in a bit of ad-hoc way as it is not locating Coq libraries]

Reply:
I mean you're getting an "unknown option -coqlib" error which needs fixing.

Reply:
I mean `coq_makefile` should be fixed.

Reply:
Anyways the point is that this kind of setup cannot both work with relative and absolute paths unless you do a hack; in case of a relative path you need to adjust the path, in case of an absolute one you don't.

Now it works just by chance.